src/mermaid-isomorphic.ts

```diff
@@ -146,9 +158,14 @@ async function renderDiagrams({
   containerStyle,
   diagrams,
   mermaidConfig,
+  mermaidElkImport,
+  mermaidImport,
   prefix,
   screenshot
 }: RenderDiagramsOptions): Promise<PromiseSettledResult<RenderResult>[]> {
+  const mermaid: Mermaid = (await import(mermaidImport)).default
+  const mermaidElk = (await import(mermaidElkImport)).default
```

Comment:
Did you read https://github.com/remcohaszing/mermaid-isomorphic/issues/15#issuecomment-2948449790? Notably this part:

> I lean towards a solution where a user can pass in a path to an arbritrary setup script. That means you can build the setup script yourself with ESBuild or something, allowing you to register any plugins, icon packs, etc with Mermaid.

Looking at this PR, I’m fine with ELK support. But what if a next user asks for ZenUML? That brings in quite the dependency tree.

Reply:
Yeah, I initially tried fully parameterizing the init script, but loading arbitrary dependencies on Playwright is a bit tricky because of the serialization and globalThis scoping. Will give it another try.

Either way, I think ELK support is a sane default to be on par with the [Mermaid Playground](https://www.mermaidchart.com/play)

Reply:
Ok, let’s just do it. Let’s add ELK support :smile: